default.nix

```diff
@@ -39,6 +39,7 @@ let
         })
         (gitTH "ormolu" "")
         (gitTH "ormolu-live" "../")
+        (gitTH "extract-hoogle-hackage-info" "../")
```

Comment:
I'm not really sure of what I did in `default.nix`, I mostly copied/adapted what was already there (but with little understanding of what it means). So a review is really needed on this file

Reply:
`gitTH` is only necessary if the project uses sth like [`gitrev`](https://hackage.haskell.org/package/gitrev), where the `.git` subfolder has to be available at compile time for Template Haskell, cf. https://github.com/tweag/ormolu/issues/342. For `extract-hoogle-hackage-info`, that is not the case, so this particular line can be removed.

The rest of the changes to `default.nix` seem fine :+1:. One thing that could be changed is that `extract-hoogle-hackage-info` currently does not compile on GHC 9.2.1 as some dependencies are not yet up-to-date. But that is not a problem, you can move it to the 8.10-only part of the file: https://gist.github.com/amesgen/78b2e76d103a3caebda1f820f8d0e642